content/pages/features.html

```diff
@@ -71,9 +71,10 @@ <h3>A Test Plan is a collection of Test Cases</h3>
                     </ul>
                 </div>
 
-                <div style="float: right; clear:right;margin-bottom: 5rem;">
-                    <img src="/images/features/testplan.png" class="margin0 shadow">
+                <div class="testplan">
+                    <img src="/images/features/testplan.png" class="testplan-img margin0 shadow">
```

Comment:
Remove testplan-img class. It adds a double 5rem margin at the botton. The wrapping div already provides this.